package.json

```diff
@@ -33,7 +33,7 @@
     "cross-env": "^7.0.2",
     "css-loader": "^4.2.0",
     "gh-pages": "^3.1.0",
-    "node-sass": "^4.14.1",
+    "sass": "^1.54.3",
```

Comment:
Nice. Unsure though if this will break older projects using und-css that are still in node-sass.

I would consider adding sass as a peer dependency as well and doing a major version of und-css (1.0.0) so yarn won't install this version into an older project by accident.